danielMunteanu/src/main/java/FilterByType.java

```diff
@@ -0,0 +1,19 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class FilterByType {
+    public static List<Order> filterOrderByItemType(List<Order> orderList) {
+        List<Order> orderResult = new ArrayList<>(); // creates a list to store the result
+
+        for (Order order : orderList) {
+            for (Item item : order.items) {
+                if (item.getType() == Type.TOY) {
```

Comment:
in case you will have a list of items that contains more then 1 clothes, then the oreder will be added twice. Find out a way in order to make it to not add duplicates.

Reply:
fixed in next commit